client/transaction-pool/graph/src/validated_pool.rs

```diff
@@ -150,9 +150,16 @@ impl<B: ChainApi> ValidatedPool<B> {
 		let future_limit = &self.options.future;
 
 		debug!(target: "txpool", "Pool Status: {:?}", status);
-
 		if ready_limit.is_exceeded(status.ready, status.ready_bytes)
-			|| future_limit.is_exceeded(status.future, status.future_bytes) {
+			|| future_limit.is_exceeded(status.future, status.future_bytes)
+		{
+			debug!(
+				target: "txpool",
+				"Enforcing limits ({}/{}b ready, {}/{}b future",
```

Comment:
I think this would be better formatted as KB, no?

Reply:
sure, but it's a technical trace